### PR TITLE
header&footer

### PR DESCRIPTION
**Pull request by spartanez17**
Wtf was here?

### CODE REVIEW

File: app/components/CurrentDate/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export default class CurrentDate extends React.PureComponent {
```
Comment (screamcha) [inline app/components/CurrentDate/index.js]: Please import `PureComponent` together with React.

File: app/components/Header/header.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+$font-stack: Helvetica, sans-serif;
+$primary-color: #c0c0c0;
```
Comment (screamcha) [inline app/components/Header/header.scss]: We probably should move colors and fonts in a separate files and import them instead.

File: app/components/CurrentDate/index.js (hunk 1 above)
Comment (spartanez17) [reply app/components/CurrentDate/index.js]: za sho

Comment (mikitasavanovich) [reply app/components/CurrentDate/index.js]: @spartanez17 I want this thing to look the same across all our files. I"m importing it together with React but we can do it your way too. Up to you to decide.

**Pull request merged**